Mergin/projects_manager.py

```diff
@@ -51,14 +51,29 @@ def have_writing_permissions(self, project_name):
         writersnames = info["access"]["writersnames"]
         return username in writersnames
 
-    @staticmethod
     def open_project(project_dir):
         if not project_dir:
             return
 
         qgis_files = find_qgis_files(project_dir)
         if len(qgis_files) == 1:
             iface.addProject(qgis_files[0])
+            if self.mc.has_unfinished_pull(project_dir):
+                widget = iface.messageBar().createMessage(
+                    "Unfinished pull",
```

Comment:
I would suggest to rename this title to "Mergin", so it is clear where this message is coming from